<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow3</class>
 <widget class="QMainWindow" name="MainWindow3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>-20</y>
      <width>971</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>791</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>340</y>
        <width>321</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(233,248,255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>code ticket :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>num ticket</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>prix :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Date  :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>type</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_code">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>10</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_numero">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_prix">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>70</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_type">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>130</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>100</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_ajouter">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
padding: 5px;
border-radius: 5px;
background-color:rgb(120, 100, 130);
font-weight: bold;
}
QPushButton:hover
{
color :white;
}</string>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>170</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
padding: 5px;
border-radius: 5px;
background-color:rgb(120, 100, 130);
font-weight: bold;
}
QPushButton:hover
{
color :white;
}</string>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_9">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>170</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
padding: 5px;
border-radius: 5px;
background-color:rgb(120, 100, 130);
font-weight: bold;
}
QPushButton:hover
{
color :white;
}</string>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>340</y>
        <width>321</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(233,248,255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <widget class="QLineEdit" name="le_rech">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_rech">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>40</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
padding: 5px;
border-radius: 5px;
background-color:rgb(120, 100, 130);
font-weight: bold;
}
QPushButton:hover
{
color :white;
}</string>
       </property>
       <property name="text">
        <string>rechercher</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_tri">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>90</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
padding: 5px;
border-radius: 5px;
background-color:rgb(120, 100, 130);
font-weight: bold;
}
QPushButton:hover
{
color :white;
}</string>
       </property>
       <property name="text">
        <string>trier</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_imprimer">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>150</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
padding: 5px;
border-radius: 5px;
background-color:rgb(120, 100, 130);
font-weight: bold;
}
QPushButton:hover
{
color :white;
}</string>
       </property>
       <property name="text">
        <string>imprimer</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_rech">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>92</width>
         <height>26</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>code</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>numero</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dates</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="cb_tri">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>90</y>
         <width>92</width>
         <height>26</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>choisir</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CODE CROISSANT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CODE DECROISSANT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NUMERO CROISSANT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NUMERO DECROISSANT</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pbpdf">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>150</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
padding: 5px;
border-radius: 5px;
background-color:rgb(120, 100, 130);
font-weight: bold;
}
QPushButton:hover
{
color :white;
}</string>
       </property>
       <property name="text">
        <string>exporter pdf</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#aa0000;&quot;&gt;GESTION DES TICKETS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
